lua/neogit/lib/git/cli.lua

```diff
@@ -95,6 +95,12 @@ local configurations = {
       end,
     },
   },
```

Comment:
This is just me being fussy (sorry), can you add an empty line between sections? It gets pretty tough to read otherwise.